scripts/projectUpdater.ps1

```
@@ -117,6 +117,26 @@ $_templateMetadata =
     $_metadata.Templates |
         Where-Object { $_.folder -eq $templateName }
 
+# If the template is deprecated or broken, it cannot be updated. If it is incomplete the user should be made aware of the problem and choose to proceed or not.
+if ($_templateMetadata.status -eq "deprecated") {
+    Write-Host -ForegroundColor DarkRed "This template is deprecated in the most recent version of the Torizon IDE Extension. For details, check https://github.com/torizon/vscode-torizon-templates/blob/dev/DEPRECATED.md"
+    exit 0
```

I prefer to return an error code here.

---

Makes sense